### PR TITLE
chore: add task request template

### PR DESCRIPTION
**Pull request by Nithos**
Adding template for general development tasks based on designs or requirements that do not require a full feature

### CODE REVIEW

**Review by YohannParis — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/task-request.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+name: Task request
+about: Template for development tasks
+title: ''
+labels: ''
```
Comment (bencodrington) [inline .github/ISSUE_TEMPLATE/task-request.md:5]: You removed these quotes

```suggestion
labels:
```

In the changes above. Do we need them here? What's the correct syntax?

**Review by bencodrington — APPROVED** [review]
Looks good to me! One syntax clarification question, otherwise merge away.

Never used the code-owners feature before, seems really handy.

File: .github/ISSUE_TEMPLATE/task-request.md (hunk 1 above)
Comment (Nithos) [reply .github/ISSUE_TEMPLATE/task-request.md:5]: hmm looking into it... it was done through github generator so not sure why they are inconsistent

Comment (Nithos) [reply .github/ISSUE_TEMPLATE/task-request.md]: its just YAML underneath so can be either or unless you need to escape something... made them all consistent now

**Review by YohannParis — APPROVED** [review]
I think we should move on with it, and if they fix that Task list issue, then great.
I will make an extra effort to follow the template when creating issues.

**Pull request merged**